### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by ohskylab**
TP: https://rockabox.tpondemand.com/entity/9583

* [x] Bump version in `package.json` and `bower.json`
* [x] Update Changelog
* [x] Create release

### CODE REVIEW

**Comment by k0nG** [comment]
:+1:

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,22 +8,31 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Added
 
+- Nothing.
+
+## [2.0.0] - 2015-65-02
```
Comment (chrisvxd) [inline CHANGELOG.md]: D'oh

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,22 +8,31 @@ This project adheres to [Semantic Versioning](http://semver.org/).
...
 - [`rb-fieldset` title attribute changed to label](https://github.com/rockabox/rbx_ui_components/pull/184).
 - [`rb-ratio` component renamed `rb-ratio-display`](https://github.com/rockabox/rbx_ui_components/pull/175)
+- [`rb-scrollspy` now animates](https://github.com/rockabox/rbx_ui_components/pull/183). Part of `angular-strap` integration.
 - [`rb-scrollspy` now builds top level list from `items` property not `sections`](https://github.com/rockabox/rbx_ui_components/pull/172)
-- [`rb-scrollspy` now animates](https://github.com/rockabox/rbx_ui_components/pull/183). As part of `angular-strap` integration.
-- [`rb-scrollspy` now uses `angular-scroll` instead of `angular-strap` scrollspy](https://github.com/rockabox/rbx_ui_components/pull/183). To enable observation of scroll positioning in custom elements such as modals.
-- [`rb-text-control` and `rb-datetime-control` title attribute changed to label](https://github.com/rockabox/rbx_ui_components/pull/179).
+- [`rb-scrollspy` now uses `angular-scroll` instead of `angular-strap` scrollspy](https://github.com/rockabox/rbx_ui_components/pull/183). Enable observation of scroll positioning in custom elements (e.g. modals).
```
Comment (chrisvxd) [inline CHANGELOG.md]: `Enable` -> `Enables`

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,22 +8,31 @@ This project adheres to [Semantic Versioning](http://semver.org/).
...
+- [`rb-text-control` and `rb-datetime-control` title attribute changed to label](https://github.com/rockabox/rbx_ui_components/pull/179)
```
Comment (chrisvxd) [inline CHANGELOG.md]: Should the attrs use backticks?

File: CHANGELOG.md [hunk 4]
```diff
@@ -8,22 +8,32 @@ This project adheres to [Semantic Versioning](http://semver.org/).
...
+- [`rb-scrollspy` now animates](https://github.com/rockabox/rbx_ui_components/pull/183). Part of `angular-strap` integration.
 - [`rb-scrollspy` now builds top level list from `items` property not `sections`](https://github.com/rockabox/rbx_ui_components/pull/172)
-- [`rb-scrollspy` now animates](https://github.com/rockabox/rbx_ui_components/pull/183). As part of `angular-strap` integration.
-- [`rb-scrollspy` now uses `angular-scroll` instead of `angular-strap` scrollspy](https://github.com/rockabox/rbx_ui_components/pull/183). To enable observation of scroll positioning in custom elements such as modals.
-- [`rb-text-control` and `rb-datetime-control` title attribute changed to label](https://github.com/rockabox/rbx_ui_components/pull/179).
+- [`rb-scrollspy` now uses `angular-scroll` instead of `angular-strap` scrollspy](https://github.com/rockabox/rbx_ui_components/pull/183). Enables observation of scroll positioning in custom elements (e.g. modals).
+- [`rb-text-control` and `rb-datetime-control` title attribute changed to label](https://github.com/rockabox/rbx_ui_components/pull/179)
```
Comment (chrisvxd) [inline CHANGELOG.md]: More attrs that need backpacking

**Comment by ohskylab** [comment]
:rocket: release: https://github.com/rockabox/rbx_ui_components/releases/tag/2.0.0

**Comment by k0nG** [comment]
MOG

**Pull request merged**